common/djangoapps/student/management/commands/bulk_unenroll.py

```diff
@@ -67,7 +78,13 @@ def unenroll_learners(self, course_id, username=None):
         if username:
             enrollments = enrollments.filter(user__username=username)
 
-        for enrollment in enrollments:
-            enrollment.update_enrollment(is_active=False, skip_refund=True)
-            logger.info("User [{}] have been successfully unenrolled from the course: {}"
-                        .format(enrollment.user.username, course_key))
+        logger.info("Processing [{}] with [{}] enrollments.".format(course_id, enrollments.count()))
+
+        if self.commit:
+            for enrollment in enrollments:
+                enrollment.update_enrollment(is_active=False, skip_refund=True)
+                logger.info(
+                    "User [{}] have been successfully unenrolled from the course: {}".format(
+                        enrollment.user.username, course_key
+                    )
+                )
```

Comment:
Why is it needed? What benefit will it provide because the main logic is behind the commit flag?

Reply:
Adds the ability to dry-run the command. If the `commit` flag isn't passed, it will not update the enrollment. 
The log on L#81 will log information about course-id and number of enrollments.

The benefit is that we can dry-run about the course ids and their enrollment count before running with commit. We can assess errors prior to making actual enrollment changes.